src/api/java/com/minecolonies/api/util/InventoryUtils.java

```diff
@@ -133,7 +133,7 @@ public static Item getItemFromBlock(final Block block)
     @NotNull
     public static Boolean isItemStackEmpty(@Nullable final ItemStack stack)
     {
-        return stack == null || stack == EMPTY || stack.getCount() <= 0;
+        return stack == null || stack == EMPTY || stack.getItem() != null || stack.getCount() <= 0;
```

Comment:
stack.getItem() == null cannot happen
If it happens it is not our fault and Forge will throw a very big warning.
That is why i left it out in the first place.